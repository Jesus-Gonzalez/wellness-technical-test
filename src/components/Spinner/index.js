export { Spinner } from './Spinner.template'
