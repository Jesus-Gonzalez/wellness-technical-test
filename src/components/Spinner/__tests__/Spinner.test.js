import React from 'react'
import renderer from 'react-test-renderer'

import { Spinner } from '../Spinner.template'

describe('components', () => {
  describe('Spinner', () => {
    it('should render', () => {
      const Component = renderer.create(<Spinner />)

      expect(Component).toMatchSnapshot()
    })
  })
})
