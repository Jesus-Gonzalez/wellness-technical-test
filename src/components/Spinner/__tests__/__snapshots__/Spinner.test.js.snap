// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Spinner should render 1`] = `
<div
  className="spinner"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
